README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## __Dominique Janvier__
+
+![_Headshot taken by my wonderful friend Amy Tian_](https://github.com/djanvier/hw01/blob/master/DominiqueHeadshot.jpg)
```

Comment:
Also, you can shorten the link to just `![_Headshot taken by my wonderful friend Amy Tian_](DominiqueHeadshot.jpg)` since the file exists in the same folder as the `README.md`